plugin.audio.navahang/main.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#  Copyright 2019 Amir Ammari
+#
+#  This file is part of the Navahang kodi plugin.
+#
+#  This plugin is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This plugin is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this plugin.  If not, see <http://www.gnu.org/licenses/>.
+
+#
+# Imports
+#
+from future import standard_library
+standard_library.install_aliases()
+from os.path import join
+from sys import argv
+import json
+from time import gmtime, strftime, strptime
+import urllib.error
+from urllib.request import urlopen
+from urllib.parse import parse_qs, urlparse, quote_plus
+from xml.dom.minidom import parseString
+from xbmc import translatePath
+import xbmcaddon
+from xbmcgui import Dialog, ListItem
+from xbmcplugin import addDirectoryItem, endOfDirectory
+
+ADDON = xbmcaddon.Addon()
+ADDONNAME = ADDON.getAddonInfo('name')
+ADDONID = ADDON.getAddonInfo('id')
+ADDONVERSION = ADDON.getAddonInfo('version')
+CWD = ADDON.getAddonInfo('path').decode("utf-8")
+LANGUAGE = ADDON.getLocalizedString
+
+DataLink = 'http://navahang.me/webservice/GetMp3?type=Featured&from=0&to=50'
+
+class LoadLister:
+
+        def __init__(self, url):
+                self.url = url
+
+        def addLink(self, name, url, image = '', info = {}, totalItems = 0):
+                item = ListItem(name.encode('utf-8'), iconImage = image, thumbnailImage = image)

```

Comment:
iconImage and thumbnailImage kwargs are deprecated. Please use only `listitem.setArt`

Reply:
```suggestion
                item = ListItem(name).setArt({ 'poster': image, 'banner': image})
```

Reply:
```suggestion
                item = ListItem(name).setArt({ 'poster': image, 'banner': image})
```